src/applications/gqrx/remote_control.cpp

```diff
@@ -328,9 +328,6 @@ void RemoteControl::setReceiverStatus(bool enabled)
 /*! \brief Convert mode string to enum (DockRxOpt::rxopt_mode_idx)
  *  \param mode The Hamlib rigctld compatible mode string
  *  \return An integer corresponding to the mode.
- *
- * Following mode strings are recognized: OFF, RAW, AM, FM, WFM,
- * WFM_ST, WFM_ST_OIRT, LSB, USB, CW, CWL, CWU.
```

Comment:
Why remove this?

Reply:
Sorry, I forgot why, but yes, it's not be removed